src/renderer/components/__tests__/cronjob.store.test.ts

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
All tests sits in parent `/components/` folder until this #3481  won't be resolved. Otherwise it's hard run (read impossible) tests locally.